dotcom-rendering/src/lib/rootStyles.ts

```diff
@@ -0,0 +1,49 @@
+import { css, type SerializedStyles } from '@emotion/react';
+import {
+	focusHalo,
+	palette as sourcePalette,
+} from '@guardian/source-foundations';
+import { paletteDeclarations } from '../palette';
+
+/**
+ * Global styles for pages:
+ * - colour-scheme aware palette declarations
+ * - `:focus` halo styles
+ * - `::selection` styles
+ */
+export const rootStyles = (
+	format: ArticleFormat,
+	darkModeAvailable: boolean,
+): SerializedStyles => css`
+	:root {
+		/* Light palette is default on all platforms */
+		${paletteDeclarations(format, 'light')}
+		body {
+			color: ${sourcePalette.neutral[7]};
+			background: ${sourcePalette.neutral[100]};
+		}
+		/* Indicate whether UI can adapt https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme */
+		color-scheme: ${darkModeAvailable ? 'light dark' : 'only light'};
```

Comment:
@alisonatwork I wonder if it's this line that prevents Chromium forced dark mode to kick in.

As the feature you're relying on is a [browser-specific experimental flag](https://developer.chrome.com/docs/web-platform/chrome-flags?hl=en#understand_the_risks), it's not something we can reliably or consistently test against in development, so thanks for highlighting it here.

Reply:
That's the one, thank you! If I go into developer tools and turn off exactly that line, then forced dark mode works as expected. I think in the long term your `/opt/in/dark-mode-web` feature will be the best option, but for now due to some still blacked-out/unreadable text with the in-progress work, the Chromium force dark flag provides a nicer user experience, as long as this `color-scheme` line is disabled.

For some context, I noticed the change immediately this morning using Cromite browser (https://github.com/uazo/cromite/) on my phone because that browser has the dark mode flag on by default, with a user toggle in the UI to disable it for sites where it breaks. I think a few other Chromium-derived browsers might have the same functionality enabled by default, although we are probably a much smaller group than the plain old Chrome users.

Reply:
Fix in #11379 — but unlikely to be merged until after the bank holiday

Reply:
No probs, enjoy the weekend! I've just put some userscript overrides on my devices to get me through. Thanks again for the quick response.

Reply:
Should be out in 15-20 minutes

Reply:
Looks good on my work laptop without the userscripts, thanks again for the quick response!